docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -2,88 +2,215 @@
+  You can [register a listener to monitor GATT connection changes](#receive-notifications-on-connection-state-changes).
+
+- Check ATT MTU of a connection with a remote GATT client
+
+  You can [read ATT MTU of a connection with a remote client](#access-connections-att-mtu).
+
+- Manage services in the local GATT server
+
+  You can [register](#register-services) and [unregister](#unregister-services) services in the local GATT server.
+
+- Notify remote GATT clients about changes of characteristic's value
+
+  You can [send notifications and indications about changes of characteristic's value](#send-notifications-about-characteristics-value-changes-to-the-clients) to remote GATT clients.
+
+- Enable reading and writing values of characteristics and descriptors registered in the local GATT server
+
+  You can [register callbacks to respond to read and write value requests from server clients](#respond-to-read-and-write-value-requests-from-server-clients).
 
 ## Prerequisites
 
-To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/application.html) and [wearable](../../api/latest/device_api/wearable/tizen/application.html) applications) and Bluetooth (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html) and [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html) applications) APIs, the application has to request permission by adding the following privileges to the `config.xml` file:
+To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/application.html), [wearable](../../api/latest/device_api/wearable/tizen/application.html), and [tv](../../api/latest/device_api/tv/tizen/application.html) applications) and Bluetooth (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html), and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html) applications) APIs, the application has to request permission by adding the following privileges to the `config.xml` file:
 
 ```
 <tizen:privilege name="http://tizen.org/privilege/application.launch"/>
 <tizen:privilege name="http://tizen.org/privilege/bluetooth"/>
 ```
-
-## Managing the Local Bluetooth Adapter
-
-You can enable or disable the local Bluetooth adapter, and set the device name using the system-provided service through the `ApplicationControl` interface (in [mobile](../../api/latest/device_api/mobile/tizen/application.html#ApplicationControl) and [wearable](../../api/latest/device_api/wearable/tizen/application.html#ApplicationControl) applications).
+## Handle UUIDs and binary data
+
+With Bluetooth API, you can handle UUIDs and binary data.
+
+### Handle UUIDs
+According to the [Bluetooth Core Specification](https://www.bluetooth.com/specifications/bluetooth-core-specification/), UUIDs that are used to represent Bluetooth objects can take the following 3 forms:
```

Comment:
Please change 3 to three:

According to the [Bluetooth Core Specification](https://www.bluetooth.com/specifications/bluetooth-core-specification/), UUIDs that are used to represent Bluetooth objects can take the following three forms:

Reply:
Done